doc/api/cli.md

```diff
@@ -2058,6 +2058,15 @@ Enables report to be generated upon receiving the specified (or predefined)
 signal to the running Node.js process. The signal to trigger the report is
 specified through `--report-signal`.
 
+### `--report-exclude-env`
```

Comment:
I thought we had a lint rule/test for keeping these in alphabetical order 😞. (Non-blocking nit as it doesn't fail any existing linting/tests.)

Reply:
I'll do it in a follow-up PR if you don't mind. It's easier to land a doc-only PR :)